rest/route/service.go

```diff
@@ -39,6 +39,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	editRoles := RequiresSuperUserPermission(evergreen.PermissionRoleModify, evergreen.RoleModify)
 	viewTasks := RequiresProjectPermission(evergreen.PermissionTasks, evergreen.TasksView)
 	editTasks := RequiresProjectPermission(evergreen.PermissionTasks, evergreen.TasksBasic)
+	// editAnnotations := RequiresProjectPermission(evergreen.PermissionTasks, evergreen.TasksAnnotation)
```

Comment:
commenting out because the PUT routes aren't finished yet so this won't be used yet